packages/shared/package.json

```diff
@@ -53,5 +53,8 @@
     "prettier": "^1.19.1",
     "stylelint": "^10.1.0",
     "stylelint-selector-bem-pattern": "^2.1.0"
+  },
+  "dependencies": {
+    "react": "^16.13.1"
```

Comment:
do we need react now in shared?

Reply:
Since we need to use the context in shared (explained in below comment) it is necessary to use react as a dependency now.